### PR TITLE
Add installation using conda.

### PR DESCRIPTION
**Pull request by rmax**
Package provided by conda-forge community. See:
https://github.com/conda-forge/eli5-feedstock#about-eli5

### CODE REVIEW

**Comment by lopuhin** [comment]
@rolando this looks great! Do new releases of eli5 appear automatically on conda, or something needs to be done?

**Comment by rolando** [comment]
The recipe requires a PR https://github.com/conda-forge/eli5-feedstock#updating-eli5-feedstock

The most simple case is when only the version and file hash changes. This implies no dependencies changes. When dependencies it requires to update the corresponding section. Here is an example: https://github.com/conda-forge/scrapy-feedstock/pull/13/files

There are discussions about automating this but nothing established yet.

Number of downloads can be seen here: https://anaconda.org/conda-forge/eli5/files

Feel free to make a PR and add yourself as a recipe maintainer.

**Comment by rth** [comment]
I think this can be merged. 

> There are discussions about automating this but nothing established yet.

PR are now automatically made on conda-forge by a bot on news PyPi releases. Though sometimes they still need some work e.g. https://github.com/conda-forge/eli5-feedstock/pull/5

**Comment by lopuhin** [comment]
@rth thanks for the heads up. I checked on https://anaconda.org/conda-forge/eli5/files and it seems that latest release there is 0.8, while latest on PyPI is 0.8.1, and OS-specific versions are even older than that (although eli5 is pure python, but I'm don't know conda enough to know if this would give an old version).
Will also check PR you referenced, thanks.

**Comment by lopuhin** [comment]
@rth ah now I see that you're working on bringing eli5 up to date on conda, thank you 👍

**Comment by rth** [comment]
The latest eli5 v0.8.1 should now be available on conda forge.

**Comment by lopuhin** [comment]
I checked that the install command added in the PR works, and gives eli5 0.8.1 👍 
Merging
